docs/tails_guide.rst

```diff
@@ -35,13 +35,11 @@ The `Tails website <https://tails.boum.org/>`__ has detailed and up-to-date
 instructions on how to download and verify Tails, and how to create a Tails USB
 stick. Here are some links to help you out:
 
--  `Download and verify the Tails .iso`_
--  `Install onto a USB stick or SD card`_
+-  `Download and verify the Tails .iso and install onto a USB stick or SD card`
```

Comment:
This line just needs the trailing underscore at the end of the link text, after the closing backquote.